### DISCUSSION TITLE
User-Defined Match File Location Assignment

### DISCUSSION THREAD

**Discussion by qalisto** [espanso/espanso#1817] [Q&A]
How can I CONFIG Espanso so that I can load/run the Operating files from one Drive, but "call" the **Replace** data from any other location (including a USB stick)?  

For context, I am running two machines, Linux and Windows (via a KVM).  I want to use the same Replace data across both platforms.

I don't see an obvious place in **default.yml** to make such adjustments.  

If there is a "script line" that I need to add to **default.yml** for each OS's main installation DIR, to properly tell the _App_ where to _find_ the **"Match"** files, please advise.

(I am trying to see if one USB portable installation will work, but I suspect that a Core installation for each OS will be less problematic.)

Thanks in advance to whomever can advise.

-qalisto

**Comment by smeech** [comment]
Is this a duplicate of #1173?
Currently this may only be possible via symbolic links, outside espanso?

**Reply by qalisto** [reply]
I think I tried this and it was problematic due to different methods of symlink recognition cross-platform.

Can revisit with more depth.   Thx.
